web/src/components/core/Section.jsx

```diff
@@ -52,6 +52,8 @@ import { ValidationErrors } from "~/components/core";
  * @property {string} [path] - Path where the section links to.
  *  when user clicks on the title, used for opening a dialog.
  * @property {boolean} [loading] - Whether the section is busy loading its content or not.
+ * @property {string} [className] - Class name for section html tag.
+ * @property {string} [sectionId] - Id of the type of the section, software, product, storage, ...
```

Comment:
NP: If you don't mind, we can just call it `id` here. It's ok `sectionId` in ValidationErrors and other components, but here it makes more sense just `id`. Up to you :)

Also, 

```suggestion
 * @property {string} [id] - Id of the section ("software", "product", "storage", ...)
```